src/components/confirm/confirm.js

```diff
@@ -7,6 +7,7 @@ function useNativeConfirm() {
     // webOS seems to block modals
     // Tizen 2.x seems to block modals
     return !browser.web0s
+        && !(browser.tizenVersion && browser.tizenVersion >= 8)
         && !(browser.tizenVersion && browser.tizenVersion < 3)
```

Comment:
```suggestion
        && !(browser.tizenVersion && (browser.tizenVersion < 3 || browser.tizenVersion >= 8))
```

Reply:
Can I have a question:
Why do we use NativeAlert at all?

With this modification we wont use it before 4 and after 8. 
And most probably nobody will check it  after 8.X that the function is corrected by Samsung so it will remain there forever.
Simple we can use this: `&& !(browser.tizenVersion)` and all version will be the same.

Reply:
> Why do we use NativeAlert at all?

Tbh, I ask the same question myself. 😅
It is inherited from Emby. 🤷‍♂️

> Simple we can use this: `&& !(browser.tizenVersion)` and all version will be the same.

`!browser.tizen` is shorter.